src/client/content_cao.cpp

```diff
@@ -1408,11 +1411,7 @@ void GenericCAO::processMessage(const std::string &data)
 		m_position = readV3F32(is);
 		m_velocity = readV3F32(is);
 		m_acceleration = readV3F32(is);
-
-		if (std::fabs(m_prop.automatic_rotate) < 0.001f)
```

Comment:
@SmallJoker Should this if-else block be restored as well?

Reply:
No, else `set_rotation` would not work any more.

Reply:
Interesting... Ok, I've restored the condition.